x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestUtils.java

```diff
@@ -310,7 +310,13 @@ public enum Type {
         SCALED_FLOAT(s -> s == null ? null : scaledFloat(s, "100"), Double.class),
         KEYWORD(Object::toString, BytesRef.class),
         TEXT(Object::toString, BytesRef.class),
-        IP(StringUtils::parseIP, BytesRef.class),
+        IP(
```

Comment:
IPs are loaded as strings in CsvTests and as BytesRef from Spec tests

Reply:
I suppose we should pick one, but this is fine for now. I think we use BytesRef most other places but whatever works is fine by me.